cmd/gpu_plugin/render-device.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
```

Comment:
we could add llicense here explicitly if this is meant to be copied?

Reply:
Same as in other scripts:
```
# Copyright 2021 Intel Corporation.
#
# SPDX-License-Identifier: Apache-2.0
```
?

Reply:
+1 to me